pkgs/by-name/ra/rastertoezpl/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  cups,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rastertoezpl";
+  version = "1.1.12";
+
+  src = fetchurl {
+    url = "https://godex.s3-accelerate.amazonaws.com/tWHDYruxWFM98frjWNTE,g.file?v01";
+    name = "${pname}-${version}-source.tar.gz";
+    hash = "sha256-iBhM8Mht/XncWU75cd485WK5GZtJNv78yMsFsD0eKWQ=";
+  };
+
+  buildInputs = [ cups ];
+
+  configureFlags = [
+    "--datarootdir=${placeholder "out"}/share"
+    "PPDDIR=${placeholder "out"}/share/cups/model"
+  ];
+
+  postInstall = ''
+    mkdir -p $out/lib/cups/filter
+    ln -s $out/libexec/rastertoezpl/rastertoezpl $out/lib/cups/filter/rastertoezpl
```

Comment:
To save some storage space, it would be worthwhile to gzip the ppd files, e.g. with `gzip -9n ${placeholder "out"}/share/cups/model/godex/*.ppd` in `postInstall`.  This reduces the package size from 21 MiB to 4 MiB on my machine.